src/kaltura-live-plugin.tsx

```diff
@@ -276,6 +276,10 @@ export class KalturaLivePlugin extends KalturaPlayer.core.BasePlugin implements
     });
   }
 
+  private _getGuiAreaNode = () => {
+    return this.player.getView().parentNode?.parentNode?.querySelector('.playkit-gui-area') || null;
```

Comment:
@semarche-kaltura how about using `state.config.targetId` like we did for overlayPortal component instead of `this.player.getView().parentNode?.parentNode?` ? if the hierarchy of the elements will be changed, it can break this, right?